tests/verify_object_limits.erl

```diff
@@ -128,3 +132,49 @@ verify_sibling_limits(C, Node1) ->
     lager:info("Result when too many siblings : ~p", [Res]),
     ?assertMatch({error,_},  Res),
     ok.
+
+verify_readrepair_ignore_max_size(C, Node1) ->
+    % Add intercept to treat all vnode puts as readrepairs
+    Intercept = {riak_kv_vnode, [{{put, 6}, put_as_readrepair},{{coord_put,6}, coord_put_as_readrepair}]},
+    ok = rt_intercept:add(Node1, Intercept),
+    timer:sleep(100),
```

Comment:
Yes you're right sorry. This was added as a debug step when the test was failing for me for an unrelated reason. I saw some sleeps in intercept adds in other tests and was trying it out. I agree, it's not necessary and can be removed.